examples/jsm/loaders/GLTFLoader.js

```diff
@@ -284,15 +284,15 @@ class GLTFLoader extends Loader {
 
 	parse( data, path, onLoad, onError ) {
 
-		let content;
+		let json;
 		const extensions = {};
 		const plugins = {};
 
 		if ( typeof data === 'string' ) {
 
-			content = data;
+			json = JSON.parse(data);
```

Comment:
Spaces after and before brackets are needed, for example `json = JSON.parse( data );`.

Please refer to the coding style guide.

https://github.com/mrdoob/three.js/wiki/Mr.doob's-Code-Style%E2%84%A2

Reply:
Fixed